workspaces/ui-v2/src/optic-components/pages/diffs/ReviewEndpointDiffPage/ReviewEndpointDiffPage.tsx

```diff
@@ -41,9 +42,9 @@ const useRedirectForDiffCompleted = (shapeDiffs: IInterpretation[]) => {
 };
 
 type ReviewEndpointDiffPageProps = {
-  endpoint: IEndpoint;
+  endpoint: RenderedDiffHeaderProps['endpoint'];
```

Comment:
Inverted this such that we are inheriting off the component we are importing (to avoid circular imports)